HackIllinois/ViewControllers/HIBaseViewController.swift

```diff
@@ -180,20 +180,24 @@ extension HIBaseViewController: NSFetchedResultsControllerDelegate {
         case .delete:
             guard let deleteIndexPath = indexPath else { return }
             tableView?.deleteRows(at: [deleteIndexPath], with: .fade)
-            if let tableView = tableView {
-                DispatchQueue.main.async {
-                    UIView.setAnimationsEnabled(false)
-                    UIView.animate(withDuration: 0.0, animations: tableView.reloadData) { _ in
-                        UIView.setAnimationsEnabled(true)
-                    }
-                }
-            }
+//            if let tableView = tableView {
```

Comment:
Add a comment of how this is the old way of forcing refresh to stop index path conflicts in table which caused it to crash, reference the part of code where it's done now in the comment too